src/main/java/com/team4/testingsystem/controllers/AnswerController.java

```diff
@@ -0,0 +1,51 @@
+package com.team4.testingsystem.controllers;
+
+import com.team4.testingsystem.dto.AnswerRequest;
+import com.team4.testingsystem.entities.Answer;
+import com.team4.testingsystem.services.AnswerService;
+import io.swagger.annotations.ApiOperation;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/answer")
+public class AnswerController {
```

Comment:
Do we really need a separate controller for answers?

Reply:
I initially misunderstood what this service is for

Reply:
Actually, you just need to have an entity here. It will store answers in the DB. At the same time, all the logic connected to it will be done in another services (question and answerOptions)